plugins/kubernetes/app/models/kubernetes/release_doc.rb

```diff
@@ -86,10 +86,15 @@ def store_resource_template
     end
 
     def add_pod_disruption_budget
-      min_available = nil
-      raw_template.each do |t|
-        min_available ||= t.dig(:metadata, :annotations, :"samson/minAvailable")
-      end
+      min_available_key = [:metadata, :annotations, :"samson/minAvailable"]
+      match_labels_key = [:spec, :selector, :matchLabels]
```

Comment:
shouldn't these be plural as in `min_available_keys`

Reply:
yeah ... renamed to _path